### DISCUSSION TITLE
"Like ClickOnce" - how?

### DISCUSSION THREAD

**Discussion by SvenAelterman** [Squirrel/Squirrel.Windows#1761] [Q&A] [answered]
If one was interested in truly using Squirrel to replace ClickOnce, is it possible to provide a link on a webpage that opens the app while providing input arguments?

**Comment by anaisbetts** [comment, marked as answer]
This isn't possible, ClickOnce does this by cheating and including binaries in the OS to do this.

And to be fair, I said it was _like_ ClickOnce! Not "feature identical" 😅

**Reply by robmen** [reply]
It is possible by "squirreling" (pun intended 😄) away command-line arguments in the filename. This technique has its limits but is quite usable for many scenarios. It is something I'd like to add in a "Squirrel v3".

ClickOnce does cheat by being included in the framework itself.
